common-custom-user-data-gradle-plugin/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'common-custom-user-data-gradle-plugin'
```

Comment:
I suggest to also enable (local) build caching via gradle.properties.

Reply:
Good point, will do.